crossing-training.lic

```diff
@@ -642,8 +643,13 @@ class CrossingTraining
   end
 
   def train_athletics
-    if DRSkill.getrank('Athletics') >= 400
+    if @climbing_target
       return if @researching
+      walk_to(@climbing_target['id'])
+      climb_practice
+    elsif DRSkill.getrank('Athletics') >= 400
+      return if @researching
+      @climbing_target = get_data('athletics').practice_options['undergondola_branch']
```

Comment:
yea that seems like a huge mistake, maybe climb practice method can just take the target as an argument and we avoid setting `@climbing_target` for the user